sdk/src/main/java/com/podio/sdk/provider/FileProvider.java

```diff
@@ -21,16 +21,16 @@
 import com.podio.sdk.domain.File;
 
 /**
- * This class provides methods to access {@link com.podio.sdk.domain.File} objects from the API.
+ * This class provides methods to access Files API area.
  *
  * @author László Urszuly
  */
 public class FileProvider extends Provider {
 
-    static class Path extends Filter {
+    public static class FileFilter extends Filter {
 
-        protected Path() {
-            super("file/v2");
```

Comment:
I looked in our API code and "file/v2" and "file" both point at the same method and I think it looks nicer with just "file" than "file/v2"